pkg/network/ebpf/c/https.h

```diff
@@ -16,7 +16,7 @@ static __always_inline void https_process(conn_tuple_t *t, void *buffer, size_t
     http_transaction_t http;
     __builtin_memset(&http, 0, sizeof(http));
     __builtin_memcpy(&http.tup, t, sizeof(conn_tuple_t));
-    read_into_buffer((char *)http.request_fragment, buffer, len);
+    http.fragment_sz = read_into_buffer((char *)http.request_fragment, buffer, len);
```

Comment:
you will need to update http_begin_request() as well to copy fragment_sz to the actual http transaction that will be sent to the userspace later